src/Arcus.Templates.Tests.Integration/appsettings.local.json

```diff
@@ -1,7 +1,8 @@
 {
   "Arcus": {
     "Api": {
-      "BaseUrl": "http://localhost:5000/api/v1"
+      "BaseUrl": "http://localhost:5000/api/v1",
+      "ProjectPath": "..\\..\\..\\..\\Arcus.Templates.WebApi\\"
```

Comment:
A little help? I need the path to the project/nuget

Reply:
Hm interesting, will have to take a look at this after my holiday. Not sure what the required value is at first glance

Reply:
Is what improved; now we can use the build sources env variable in a CI build; and a simpler path in a local app settings file.

Reply:
👌